lib/Net/Google/Drive/Simple/V3.pm

```diff
@@ -9,24 +9,33 @@ use parent qw< Net::Google::Drive::Simple::Core >;
 use URI ();
 use URI::QueryParam ();
 use File::Basename ();
+use JSON qw< from_json >;
 use Log::Log4perl qw(:easy);
 
 use constant {
     'HTTP_METHOD_GET'    => 'GET',
     'HTTP_METHOD_POST'   => 'POST',
+    'HTTP_METHOD_PUT'    => 'PUT',
     'HTTP_METHOD_PATCH'  => 'PATCH',
     'HTTP_METHOD_DELETE' => 'DELETE',
 
+    'HTTP_CODE_OK'     => 200,
+    'HTTP_CODE_RESUME' => 308,
```

Comment:
maybe reuse the `HTTP_CODE_RESUME` constant from Simple::Core? or only define it in a single location?